dataprofiler/data_readers/data_utils.py

```diff
@@ -12,14 +13,14 @@
 from chardet.universaldetector import UniversalDetector
 
 from .. import dp_logging
-from .filepath_or_buffer import FileOrBufferHandler, is_stream_buffer
```

Comment:
this being removed means the file calling `data_utils.is_stream_buffer` can't use it.